code/rounds/InProgressRound.cs

```diff
@@ -141,5 +152,12 @@ private void AssignRoles()
                 }
             }
         }
+
+        private void LoadPostRound(BaseRole winningRole)
+        {
+            // TODO: Consider maybe moving to PostRound
+            TTTPlayer.ClientOpenAndSetPostRoundMenu(winningRole.Name);
```

Comment:
Why just the Name? Couldn't we also pass the winningRole.Color to set it as background-color for the Header